service/src/main/resources/schema/development/schema.json

```diff
@@ -21,11 +21,11 @@
       "type": "string"
     },
     "dct:issued": {
-      "type": "date-time",
```

Comment:
Did this change because the schema version changed, so it's not supported? Or was `date-time` not correct to begin with? @tcjiang9